README.md

```diff
@@ -24,11 +24,16 @@ There are 7 modules in this package:
 - numpy
 - scipy
 - numpy-stl
+- plotly
+- Jupyter Notebook
+- python 3.8.x (64-bits) only
```

Comment:
The module actually supports from Python 3.5 to Python 3.8. The package I shared to you was built from Python 3.8, thus it only support Python 3.8